lib/ImportStatements.js

```diff
@@ -9,13 +9,15 @@ import uniqBy from 'lodash.uniqby';
 import Configuration from './Configuration';
```

Comment:
I've been thinking about editor configs and character widths. I think I'd like to keep my editor configured at 80 characters and then use the extra 20 characters that the linter rule allows as buffer space for troublesome sections of code. This allows for comments, for example, to follow a more readable line length for English but allows you to push the boundary for code when it is appropriate. Thoughts?